examples/go-xdp-counter/container-deployment/Containerfile.go-xdp-counter

```diff
@@ -1,22 +1,29 @@
-FROM golang:1.21 as gocounter-build
+ARG BUILDPLATFORM=linux/amd64
```

Comment:
Does the makefile for the examples need to be updated? I guess we can do that in a follow up

Reply:
Yes, I missed that.

Done.